networktest/emcli.go

```diff
@@ -48,6 +49,31 @@ func (cli Emcli) Send(from, to Key, amount string) (string, bool, error) {
 	return execCmdWithInput(args, KeyPwd)
 }
 
+func (cli Emcli) SendOnBehalf(from Key, to Key, signer Key, amount string) (string, error) {
+
+	//CREATE UNSIGNED TRANSACTION
+	msg, error1 := cli.CustomCommand("tx", "bank", "send", from.GetAddress(), to.GetAddress(), amount, "--generate-only")
+
+	if error1 != nil {
+		fmt.Println("ERROR!!")
```

Comment:
Test should probably be made to fail with meaningful context in output here or an error should be returned.